### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by HyperWinX** [DiscUtils/DiscUtils#283] [Q&A]
Where i can find FULL DOCUMENTATION about this lib? I moved big part of lib in my project, and now i need to use a lot of functions, but i dont know how to do it

**Comment by HyperWinX** [comment]
Also where i can find source codes of DiscUtils.Containers maybe? I need all functions to manipulate virtual hard disk images

**Reply by Olof-Lagerkvist** [reply]
DiscUtils.Containers is more or less a meta package that just references all supported container format libraries. It contains very little source code in its own. But the source code is in the repository.

**Reply by HyperWinX** [reply]
Ok, where i can find all documentation?
